shardManager/src/main/kotlin/space/votebot/shardmanager/core/ShardRegistry.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * VoteBot
+ * Copyright (C) 2019 VoteBot
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package space.votebot.shardmanager.core
+
+import mu.KotlinLogging
+
+class ShardRegistry {
+
+    private val log = KotlinLogging.logger {}
+    val shards = mutableListOf<Shard>()
```

Comment:
Is it possible to make the mutator private and the accessor public?

Reply:
Yes it should be possible like this
```Kotlin
var foo = "HII"
	private set
```